apidef/oas/server.go

```diff
@@ -58,8 +59,11 @@ func (s *Server) ExtractTo(api *apidef.APIDefinition) {
 	if s.ClientCertificates != nil {
 		s.ClientCertificates.ExtractTo(api)
 	}
+
 	if s.GatewayTags != nil {
 		s.GatewayTags.ExtractTo(api)
+	} else {
+		api.TagsDisabled = true
```

Comment:
The nil check should be the responsibiity of ExtractTo(), as the responsibility to fill apidef is there

Reply:
it is also outside in your PR, the only difference here we set `true` in `else` case because of reverse logic bool.

Reply:
Implementing as-is, leaving for code-improvement sprints.

Reply:
@furkansenharputlu actually no, the default gateway behavior is to have DisableTags=false (zero value)?